<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [EQUITY REJECTED ORDERS]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="EquityOrder Rejected Creation(failed)">
    <classes>
      <class name="Orders.Create">
        <methods>
          <include name="afterMethod"/>
          <include name="Verify_Equity_Order_Rejected_Creation" invocation-numbers="2"/>
          <include name="beforeMethod"/>
        </methods>
      </class> <!-- Orders.Create -->
      <class name="Subscriptions.Equity_Order_Subscribe">
        <methods>
          <include name="beforeMethod"/>
          <include name="afterMethod"/>
          <include name="Verify_SubscribeRejectedEquityOrder" invocation-numbers="1 2"/>
        </methods>
      </class> <!-- Subscriptions.Equity_Order_Subscribe -->
    </classes>
  </test> <!-- EquityOrder Rejected Creation(failed) -->
</suite> <!-- Failed suite [EQUITY REJECTED ORDERS] -->
